framework/sandstone.cpp

```diff
@@ -3707,19 +3709,6 @@ int main(int argc, char **argv)
     if (sApp->total_retest_count < -1 || sApp->retest_count == 0)
         sApp->total_retest_count = 10 * sApp->retest_count; // by default, 100
 
-    load_cpu_info(sApp->enabled_cpus);
```

Comment:
(GitHub doesn't let me comment on a line more than 3 lines away from a line you modified)
The thermal monitor and the MCE counter are now initialised before the CPU info is loaded. That's not an issue today, but I'd rather that was kept.

Reply:
I generally moved everything related to the execution past modifying the test lists, basically to avoid executing anythin in vain. However, yes, we can keep topology initialization at the beginning.